### DISCUSSION TITLE
Better exceptions logging

### DISCUSSION THREAD

**Discussion by renbrauge** [microsoft/p4vfs#17] [Ideas]
I ran into [this issue](https://github.com/microsoft/p4vfs/issues/4) as well, it happens quite often in my CI pipeline but never locally.
The main problem is that the way the exception is logged (and not just in this instance), doesn't actually provide any info, it'd be nice if in a case like this, the actual errors were logged

**Comment by jessk-msft** [comment]
It looks like [this issue](https://github.com/microsoft/p4vfs/issues/4) is from quiet, single flush, sync. Something like `p4vfs sync -q -m single` or `p4vfs populate`. It is the fastest and most fragile way to sync. However, I agree we should really improve the error output if this flush or the file modifications fail. I bet that the error message is being silented discarded. Probably needs a little debugging, while interupting the server connection. (it would make a good unit test)

**Comment by jessk-msft** [comment]
Note that this issue should be fixed in 1.28.0.0
The error messages during a quite single mode sync should be properly displayed now.
Regards
-Jess
